.bazelrc

```diff
@@ -40,19 +40,34 @@ common:remote-shared --remote_timeout=600
 common:remote-shared --remote_download_minimal
 common:remote-shared --jobs=100
 common:remote-shared --verbose_failures
-common:remote-shared --platforms=//platforms:linux_x86_64
-common:remote-shared --extra_execution_platforms=//platforms:linux_x86_64
```

Comment:
I think removing the targeted platform out of 'remote-shared' config broke this PR's CI.

We have a bunch of stuff using 'remote-shared'

```
grep '=remote-shared' .bazelrc
common:remote --config=remote-shared
common:remote-dev --config=remote-shared
common:probers --config=remote-shared
common:probers-dev --config=remote-shared
common:linux-workflows --config=remote-shared
```

and those would require the same target+exec platform flags.

So you might need to move these platform flags to a separate config, then add those configs into probers, probers-dev and linux-workflows to unblock CI.

Reply:
Done!